core/templates/pages/exploration-editor-page/editor-navigation/editor-navigation.component.html

```
@@ -83,9 +83,9 @@
       <div ngbDropdown class="exp-nav-dropdown-content right-container">
 
         <div *ngIf="isEditableOutsideTutorialMode()">
-          <div [ngClass]="{'disable-save-button': !connectedToInternet || !isExplorationSaveable()}">
+          <div [class.disable-save-button]="!connectedToInternet || !isExplorationSaveable()">
             <button class="btn publish-button e2e-test-save-changes-for-small-screens"
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)